portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +25,39 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  
+  private ArrayList<String> comments = new ArrayList<String>();
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Add new comment to my ArrayList of comments.
+    String newComment = request.getParameter("new-comment");
```

Comment:
This param name seems odd. I would format it the same way you format a variable name (camel case).

```
commentText
```